code/index.html

```diff
@@ -1,30 +1,140 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Why is there two HTML files that pretty much look the same?

Reply:
I created a copy as a backup before I made changes to index.html, this was in case my changes caused any problems. I was supposed to delete index - Copy.html before submitting the project